client/landing/jetpack-cloud/components/activity-card-list/index.jsx

```diff
@@ -71,22 +71,30 @@ class ActivityCardList extends Component {
 
 		return logsByDate.map( ( { date, logs: dateLogs }, index ) => {
 			return (
-				<Fragment key={ `activity-card-list__date-group-${ index }` }>
+				<div key={ `activity-card-list__date-group-${ index }` }>
 					{ showDateSeparators && (
-						<div className="activity-card-list__date">{ date && date.format( 'MMM Do' ) }</div>
+						<div className="activity-card-list__date-group-date">
+							{ date && date.format( 'MMM Do' ) }
+						</div>
 					) }
-					{ dateLogs.map( activity => (
-						<ActivityCard
-							{ ...{
-								key: activity.activityId,
-								moment,
-								activity,
-								allowRestore,
-								siteSlug,
-							} }
-						/>
-					) ) }
-				</Fragment>
+					<div className="activity-card-list__date-group-content">
+						{ dateLogs.map( activity => (
+							<ActivityCard
+								{ ...{
+									key: activity.activityId,
+									moment,
+									activity,
+									allowRestore,
+									siteSlug,
+									className:
+										activity.activityType === 'Backup'
```

Comment:
Here we could have used the helper function `isSuccessfulBackup` from utils. But with the new field `type` that you added is fine as well.